src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -344,12 +345,32 @@ export const selectSellAmountFiat = createSelector(
   },
 )
 
-export const selectDonationAmountFiat = createSelector(
+export const selectDonationBps: Selector<ReduxState, string | undefined> = createSelector(
```

Comment:
Similar to the comment above WRT vernacular granularity, this could be `selectActiveQuoteDonationBps`, making it clear that this is the bps for a quote - not the bps for a specific swapper, and not the actual bps that were donated

Reply:
I like it, went with `selectActiveQuoteDonationBps`, as it _is_ the actual quote donation bps. Naming is hard!

https://github.com/shapeshift/web/pull/4902/commits/d7641e10ca68c0fa631a463acb55c5c0953be8c8